packages/flutter_tools/templates/app/integration_test/driver.dart

```diff
@@ -0,0 +1,18 @@
+// This file is provided as a convenience for running integration tests via the
+// flutter drive command.
+//
+// flutter drive -t integration_test/app_test.dart --driver integration_test/driver.dart
```

Comment:
I'd rather avoid having the extra directory here that might encourage people to create driver tests we'd rather they not.

TBH, I'd rather avoid having this file at all if we can come up with som ebetter way to have the tool (or a tool) run these :)

Reply:
That makes sense. If we don't have a tool we should definitely update the documentation to match, but I'd very much like to avoid the need for this file as well 🙂. I'll try to come up with a rough prototype next week to see if it's feasible to include an executable in `package:integration_test` (don't think any packages in flutter/flutter/packages do so today)